src/components/storage/ReclaimSpaceModal.scss

```diff
@@ -33,6 +26,102 @@
     font-size: var(--pf-v5-global--FontSize--sm);
     color: var(--pf-v5-global--warning-color--100);
   }
+
+  .pf-v5-c-table__td {
+    padding-block: var(--pf-v5-global--spacer--sm);
+    align-items: center;
+  }
+
+  .reclaim-actions {
+    // Actions should not have a gap
+    gap: 0;
+    // Actions should "eat into" TD padding so they're aligned appropriately
+    margin: calc(-1 * var(--pf-v5-global--spacer--xs)) calc(-1 * var(--pf-v5-global--spacer--sm));
+
+    // Fix the button size of actions
+    .pf-v5-c-button {
+      padding: var(--pf-v5-global--spacer--xs) var(--pf-v5-global--spacer--sm);
+    }
+  }
+
+  // Bind indent level variable to classes (start indenting on level 2)
+  @for $i from 1 through 10 {
+    .reclaim-space-modal-device-level-#{$i} {
+      --indent: #{$i - 1};
+    }
+  }
+
+  // Small sizes
+  @media (width < $pf-v5-global--breakpoint--md) {
+    // Indent the left for smaller sizes
+    .reclaim-space-modal-table-row:not(.reclaim-space-modal-device-level-0) {
+      margin-inline-start: calc(
+        var(--indent) * var(--pf-v5-global--spacer--sm)
+      );
+    }
+
+    // Remove labels with no content on small sizes
+    td:empty {
+      display: none;
+    }
+
+    tr {
```

Comment:
Isn't this too invasive on the Patternfly table structure? 
What's the problem with the default? 
Please add a comment if keeping this.

Reply:
This isn't a PatternFly table; it's a Cockpit table which inherits from PF. As a result, we can't adjust it with PatternFly options.

But even still, if it were a plain PatternFly table: PatternFly does not officially support right aligning columns, which is completely absurd, as it's a basic thing that's been around for literal centuries in tables outside of computing and inside of computing basically everywhere else for decades. Some things, such as numbers, especially with units, really _need_ that right alignment. As do the actions at the end.

Additionally, I was testing for smaller widths, not just for mobile mode. As you can see the "before" screenshot has issues in smaller sizes. Smaller sizes show up when installing inside a VM... for example, in Cockpit Machines.

Reply:
There are books about designing tables and layout (and I've read a few, such as _The Visual Display of Quantitative Information_ by Edward Tufte).

But here's a distilled version:
https://www.csescienceeditor.org/article/best-practices-in-table-design/

An even simpler version is this rule of thumb, adapted for computer UI:
1. Right align numbers (ideally separator align if possible, but that's trickier on the web)
2. Left align names (in LTR; this flips in RTL)
3. Use the same alignment for the headings
4. Use monospaced numbers (which can either be monospace font or tabular numbers in a proportional font) 

Sadly, PatternFly doesn't do any of this by default, except defaulting to left alignment.